workshops/templates/workshops/admin_dashboard.html

```diff
@@ -39,6 +40,13 @@
     <table class="table table-striped">
     {% for event in current_events %}
     <tr>
+      {% for tag in carpentries %}
+        <td class="text-center" width="25px">
+          {% if tag in event.tags.all %}
+            {% bootstrap_tag tag.name %}
+          {% endif %}
+        </td>
+      {% endfor %}
```

Comment:
This approach yields a lot of queries.

The best would be to drop "must-be-aligned" constraint and prefetch all related tags and then loop over them.